ladybug/designday.py

```diff
@@ -222,6 +245,15 @@ class DesignDay(object):
         humidity_condition: Ladyubug HumidityCondition object
         wind_condition: Ladyubug WindCondition object
         sky_condition: Ladybug SkyCondition object
+
+    Properties:
+        * name
+        * day_type
+        * location
+        * dry_bulb_condition
+        * humidity_condition
+        * wind_condition
+        * sky_condition
```

Comment:
As noted above, this list should be extended with the properties currently listed in the `DDY` class docstring.